adapter/config/types.go

```diff
@@ -306,7 +307,7 @@ type analytics struct {
 	}
 
 	Enforcer struct {
-	// TODO: (VirajSalaka) convert it to map[string]{}interface
+		// TODO: (VirajSalaka) convert it to map[string]{}interface
```

Comment:
Is this TODO still valid

Reply:
yes I guess
It is an improvement we could bring in later